### PR TITLE
update links to old API

### PR DESCRIPTION
**Pull request by mickeyn**
+some docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,4 +135,4 @@ Mailing List
 ------------
 
 Our mailing list is open to all:
-[http://groups.google.com/group/cpan-api](http://groups.google.com/group/cpan-api) 
+[http://groups.google.com/group/cpan-api](http://groups.google.com/group/cpan-api)
```
Comment (oalders) [inline README.md]: I wonder if we should just delete this.  I can't remember the last time someone used the list.  I had actually forgotten that it exists.  Does it still exist?

File: lib/MetaCPAN/Script/River.pm [hunk 2]
```diff
@@ -84,8 +84,8 @@ updates our ES information.
 
 This can then be accessed here:
 
-http://api.metacpan.org/distribution/Moose
-http://api.metacpan.org/distribution/HTTP-BrowserDetect
+http://fastapi.metacpan.org/distribution/Moose
+http://fastapi.metacpan.org/distribution/HTTP-BrowserDetect
```
Comment (oalders) [inline lib/MetaCPAN/Script/River.pm]: We should include the `v1` in the urls.

File: lib/MetaCPAN/Script/Tickets.pm [hunk 3]
```diff
@@ -267,8 +267,8 @@ out ES information.
 
 This can then be accessed here:
 
-http://api.metacpan.org/distribution/Moose
-http://api.metacpan.org/distribution/HTTP-BrowserDetect
+http://fastapi.metacpan.org/distribution/Moose
+http://fastapi.metacpan.org/distribution/HTTP-BrowserDetect
```
Comment (oalders) [inline lib/MetaCPAN/Script/Tickets.pm]: We should include the `v1` in the urls.

File: README.md (hunk 1 above)
Comment (mickeyn) [inline README.md]: removed the mailing list section

File: lib/MetaCPAN/Script/River.pm (hunk 2 above)
Comment (mickeyn) [inline lib/MetaCPAN/Script/River.pm]: updated

File: lib/MetaCPAN/Script/Tickets.pm (hunk 3 above)
Comment (mickeyn) [inline lib/MetaCPAN/Script/Tickets.pm]: updated

**Pull request merged**